src/components/hero/searchBar.tsx

```diff
@@ -0,0 +1,65 @@
+import { FaSearch } from 'react-icons/fa'
+import { TiLocationArrow } from 'react-icons/ti'
+import styles from '../../styles/search.module.css'
+import { ChangeEvent } from 'react'
+import { RiArrowDropDownLine } from 'react-icons/ri'
+import FakeData from '../../api/mock-city-searchBar.json'
+import { z } from 'zod'
+
+const schema = z.object({
```

Comment:
```suggestion
const Schema = z.object({
```
Enfin moi j'aime bien les majuscule, et aussi un nom qui décrit ce que ton schéma est ça peut être cool